### DISCUSSION TITLE
[Request] Notifiarr

### DISCUSSION THREAD

**Discussion by Beel12213** [tteck/Proxmox#2984] [Ideas]
Would it be possible to add a script which installs and sets up an LXC for [Notifiarr](https://notifiarr.wiki/en/home). This connects to our other Servarr apps to update TRaSH Guides for better quality downloads along with notifications within Discord for the apps.

I've managed to manually install the app, but had to jump through a few hoops to get it working, but it would be great if you could create a script which does something similar to the below;
- bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/debian.sh)"
- apt install gnupg
- curl -s https://golift.io/repo.sh | sudo bash -s - notifiarr

Then the user needs to manually edit the below file and enter the API Key from Notifiarr.com along with create a password for the UI:
- nano /etc/notifiarr/notifiarr.conf

Can be accessed by going to: 
- http://localhost:5454

**Comment by halpcomputar** [comment]
This is one thing I've been missing as far as as an LXC that would be great to have. Notifiarr can save HOURS of time in setting up TRaSH sync, and gives really nice notificiations for just about anything you'd want to self host. High wife-approval factor with a discord channel to let her know when her new shows drop.

Quick question @Beel12213 as I've only run Notifiarr in previous setups where I was mostly docker-based: what did you give your container re: CPU, memory and storage? I'm trying to keep things as minimal as possible.

**Reply by Beel12213** [reply]
Yeah I'm using it for the same purposes, makes monitoring and the TRaSH sync so flawless. It's fantastic. Especially with the new flow charts they've added for TRaSH to auto-grab all the relevant profiles with 1 click.

It barely requires anything, so I've set it to the following;
- Cores: 1 
- Memory: 512
- Storage: 2

**Comment by tteck** [comment]
Ready for testing

https://helper-scripts.com/scripts?id=Notifiarr

**Reply by amerikiwi93** [reply]
I just installed notifiarr yesterday. I'm going to reinstall using the script and will let you know if there are any issues.

**Comment by amerikiwi93** [comment]
Tested the script today and it worked great. Only change I'd like made is the ability to add the API during the installer so I don't have to log into the LXC and modify the file.
